### PR TITLE
#1289 PCLConfig.cmake.in, OSX + homebrew

### PR DESCRIPTION
**Pull request by ahundt**
improves cmake to help deal with #1289

### CODE REVIEW

**Comment by taketwo** [comment]
Hey @fran6co you are a Mac user, do you have any opinion on this? Should we merge?

File: PCLConfig.cmake.in [hunk 1]
```diff
@@ -494,13 +494,24 @@ IF (WIN32)
 ELSE (WIN32)
 
   IF (APPLE)
+# These values for Apple could probably do with improvement.  IF (APPLE)
```
Comment (fran6co) [inline PCLConfig.cmake.in]: This looks like a typo

**Comment by fran6co** [comment]
Why is glew dependency in ```cmake/Modules/FindGLEW.cmake``` and ```PCLConfig.cmake.in```? Shouldn't it be better to just have it in ```cmake/Modules/FindGLEW.cmake```?

**Comment by jspricke** [comment]
```PCLConfig.cmake.in``` is installed along with PCL on the target system and used for cmake projects depending on PCL (and transitively on flew).

**Comment by fran6co** [comment]
I see, shouldn't it be better to avoid duplicating the code by injecting ```cmake/Modules/FindGLEW.cmake``` in it (not sure how to do that with generated files) or at least copying the code in it in ```PCLConfig.cmake.in``` as they do the same thing.

**Comment by jspricke** [comment]
`PCLConfig.cmake.in` could definitely need some cleanup. Would be great if you could prepare a patch. :)

**Comment by fran6co** [comment]
I'll see what I can do.

**Comment by taketwo** [comment]
@ahundt, could you please remove the extra line pointed by Francisco?

**Comment by ahundt** [comment]
@taketwo I think the comment is accurate, but perhaps that's better captured by a new issue?

**Comment by ahundt** [comment]
fixed the extra line

**Comment by bu2** [comment]
Dear PCL team,

Is it possible to build PCL apps on OS X 10.11 (El Capitan)

Here is an extract of the cmake output:
...
-- The following subsystems will not be built:
--   apps: No reason
...

Please find the full CMake output attached to this message.
[cmake.txt](https://github.com/PointCloudLibrary/pcl/files/40294/cmake.txt)

I installed required dependencies with Homebrew (under my home folder !).

And I am especially interested in the app in_hand_scanner.

Regards.

**Comment by fran6co** [comment]
Could you post this as a different issue? I don't think it's related to the current one.

**Comment by bu2** [comment]
Sure, here is the new issue:
https://github.com/PointCloudLibrary/pcl/issues/1441

Regards.

**Pull request merged**